README.md

```diff
@@ -79,6 +87,20 @@ Once you have `rustup` installed, then you need to install:
 
 Make sure that these binaries are in your `PATH`. After that, you should be able to build Parity Ethereum from source.
 
+or
+
+ You can build Parity Ethereum from the sources; you can launch Parity Ethereum using the binaries built for the latest releases for Linux, Mac/Homebrew and Windows or, if you’re on an Ubuntu Snappy platform, just use our Snappy App.
+ 
+ For Linux systems:
+
+Ubuntu, Debian
+
+  ```$ apt-get install build-essential cmake libudev-dev```
```

Comment:
I don't think `libudev-dev` is required anymore since we removed `hardware wallet` support